lab-michael/server.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const express = require('express');
+const morgan = require('morgan');
+const jsonParser = require('body-parser').json();
+
+const createError = require('http-errors');
+
+const RecipeApp = express();
```

Comment:
I would suggest just sticking with `app`, to keep it clean and standard for now.